robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowView.java

```diff
@@ -133,7 +134,7 @@ protected Drawable buildDrawable(int resourceId) {
   }
 
   protected String getQualifiers() {
-    return shadowOf(realView.getResources().getConfiguration()).getQualifiers();
```

Comment:
Configuration.getQualifiers() and ShadowView.getQualifiers() are not used in Robolectric and not used in our internal codebase either so I doubt they are used much if at all. I'm not sure why qualifiers are exposed on the view at all.

I agree with the sentiment about statics, I think RuntimeEnvironment.setQualifiers() is "supposed" to be a method used only internally in Robolectric and @Config(qualifiers = "...") is the correct way of doing things.

Configuration changes are handled by activities through Activity.onCongifurationChanged(Configuration) which the test writer would simulate through ActivityController.configurationChange().

We should probably be invoking activity.getResources().calcConfigChanges(configuration) in the controller also so that the underlying resources are changed so the user would actually get the landscape resources over the portrait if simulating a screen rotation for example.

Most users seem happy to create this object themselves and pass it in, but we could always have a helper method to construct a Configuration object from a qualifiers string. 

So long story short, the setQualifiers() on Configuration doesn't actually do anything nor seems to be used, so I think its vestigial code left over from way back when.....